doc/nrf/ug_gzll.rst

```diff
@@ -70,19 +65,33 @@ The radio and timer interrupt handlers should run at priority level 0 (highest p
 To avoid blocking Gazell operations, applications can run at priority level 2 or higher.
 
 You can customize Gazell at runtime for a range of different applications.
-See the :ref:`nrfxlib:gzll_api` for a list of configuration functions as well as the default and constant parameters.
+See the :ref:`nrfxlib:gzll` and :ref:`nrfxlib:gzll_api` for a list of configuration functions as well as the default and constant parameters.
+
+.. note::
 
-.. note:
    Editing the header file containing the default and constant parameters does not change their value when compiling a new project.
    These values are provided as a useful reference when making an application with the precompiled library.
 
 Gazell modes
 ************
 
-A member of a Gazell star network is either a Host or Device.
-Up to eight Devices can communicate with a single Host.
-Each Host can communicate with up to eight Devices.
-Each Device communicates to a single Host.
+Gazell has the following two modes available:
```

Comment:
Shall we change as suggested in https://github.com/nrfconnect/sdk-nrf/pull/6482/files#r819271065?
```suggestion
Gazell has the following two roles available:
```

Reply:
Yes, I agree. I will apply this change throughout, wherever it is applicable.